src/security/ldap/src/main/java/org/geoserver/security/ldap/LDAPRoleService.java

```diff
@@ -353,7 +361,7 @@ public int getRoleCount() throws IOException {
     }
 
     private String normalizeGroupName(String role) {
-        if (role.startsWith("ROLE_")) {
+        if (role.startsWith(rolePrefix)) {
             // remove standard role prefix
             role = role.substring(5);
```

Comment:
Same as above. See if there is any other place where this "5" appears and fix.

Reply:
fixed